packages/api/test/mocks/cluster/post_add.js

```diff
@@ -0,0 +1,22 @@
+// @ts-ignore not typed
+const Hash = require('ipfs-only-hash')
```

Comment:
We should probably use the importer directly, this just returns the hash of the first file...(which was good enough in nft.storage because we only had an upload blob test)